test/tools/libtesteth/TestHelper.h

```diff
@@ -166,6 +166,7 @@ void doTransactionTests(json_spirit::mValue& _v, bool _fillin);
 void doStateTests(json_spirit::mValue& v, bool _fillin);
 void doVMTests(json_spirit::mValue& v, bool _fillin);
 void doBlockchainTests(json_spirit::mValue& _v, bool _fillin);
+void doBlockchainTestsInternal(json_spirit::mValue& _v, bool _fillin);
```

Comment:
It's not really internal since it's in the header file. I'd name this one maybe just `doBlockchainTests` and another one `doBlockchainTestsWithLog` or something